spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -53,6 +54,8 @@
  */
 @Configuration
 @AutoConfigureAfter(GcpContextAutoConfiguration.class)
+@ConditionalOnProperty(value = "spring.cloud.gcp.trace.enabled", matchIfMissing = true)
```

Comment:
Does this check if `spring.cloud.gcp.trace.enabled` is `true`? Or do we need a `havingValue` attribute?
From `@ConditionalOnProperty`, it isn't clear.

Reply:
![image](https://user-images.githubusercontent.com/490484/35141947-06537c0a-fcfd-11e7-9118-6430ce00df02.png)

I find the javadoc pretty explicit. What wasn't clear?

Reply:
I meant 

```
	/**
	 * The string representation of the expected value for the properties. If not
	 * specified, the property must <strong>not</strong> be equals to {@code false}.
	 * @return the expected value
	 */
	String havingValue() default "";
```

The Javadoc you pasted is pretty clear. It seems things are OK as they are. Thanks!

Reply:
it's the class javadoc. If you think the `havingValue` javadoc could be polished, please feel free to open an issue with a suggestion.